pkg/minishift/registration/redhat_test.go

```diff
@@ -75,15 +75,7 @@ func TestRedHatRegistratorRegister(t *testing.T) {
 	}
 	commander := provision.GenericSSHCommander{Driver: d}
 	registrator := NewRedHatRegistrator(commander)
-	s.CommandToOutput["sudo subscription-manager version"] = `server type: RedHat Subscription Management`
-	if err := registrator.Register(param); err != nil {
-		t.Fatal("Distribution shouldn't have to register")
-	} else {
-		cmd := "sudo subscription-manager register --auto-attach --username foo --password foo"
-		if _, ok := s.Commands[cmd]; ok{
-			t.Fatalf("Expected command: %s", cmd)
-		}
-	}
+
```

Comment:
Actually, this is a comment for line 51-60. I like the test. I think in these cases, it is always nice to show both sides of the test. So in case we dont have 'rhel' the distribution is indeed not compatible. Especially with boolean conditions you most of the times want to check both sides.